distribution/test/itests/test-itests-ddf/src/test/java/ddf/test/itests/catalog/TestFederation.java

```diff
@@ -265,6 +278,9 @@ public void beforeExam() throws Exception {
 
     @Before
     public void setup() throws Exception {
+
+        getCatalogBundle().setDownloadRetryDelayInSeconds(1);
```

Comment:
As far as I can think, there aren't any tests (and shouldn't be any tests) that changing the delay _between_ download retries would cause an issue. As long as the retry _timeout_ isn't changed. All that the retry delay does is wait a certain amount of additional time after timing out an unresponsive connection. This _should_ just make the tests faster by removing extraneous wait time that's only there to give the network in a production environment time to potentially recuperate.